src/elm/clinical.ts

```diff
@@ -183,14 +195,15 @@ export class ConceptDef extends Expression {
   codes: any;
   display?: string;
 
-  constructor(json: any) {
+  constructor(json: ELM.ConceptDef) {
+    // @ts-ignore
```

Comment:
Same as initial comment with more explanation of ts-ignore and disabling the lint rule